app/src/main/java/org/wikipedia/page/PageFragmentLoadState.java

```diff
@@ -272,19 +266,22 @@ private void showPageOfflineMessage(@NonNull String dateHeader) {
         }
     }
 
-    private void pageLoadLeadSectionComplete(PageLead pageLead) {
+    private void createPage(@NonNull PageSummary pageSummary, @NonNull MediaList mediaList) {
+        String leadImageUrl = null, leadImageName = null;
+
         if (!fragment.isAdded()) {
             return;
         }
+        List<MediaListItem> items = mediaList.getItems("image");
 
-        Page page = pageLead.toPage(model.getTitle());
-        bridge.execute(JavaScriptActionHandler.setUpEditButtons(true, !page.getPageProperties().canEdit()));
-
+        if (items.get(0) != null) {
+            leadImageUrl = "https:" + items.get(0).getImageUrl(DimenUtil.calculateLeadImageWidth());
+            leadImageName = items.get(0).getTitle().replace("File:", "").trim();
```

Comment:
The lead image comes from `summary->thumbnail` or `summary->originalimage`.  The "first" image in the media list will almost certainly not be the lead image.